### DISCUSSION TITLE
json-schema-faker redirecting browser to /[object%20Object]

### DISCUSSION THREAD

**Discussion by scottmollon-invicara** [json-schema-faker/json-schema-faker#786] [General]
I have a case, with 0.5.3, where my React app is redirecting to /[object%20Object] when loading.
If I rollback to a much earlier version of jsf, the problem goes away.

I believe this is because of vendor.js setting the global.location.href to process.cwd() when it loads.
Why would jsf need to set the browser location?

**Comment by pateketrueke** [comment]
>  Why would jsf need to set the browser location?

There's no need actually, it's just a mistake, gonna fixit asap!

**Comment by pateketrueke** [comment]
I think this can be closed as we deployed a new version without `location` issues, thank you!
